packages/connectors/connector-aliyun-sms/src/index.ts

```diff
@@ -20,7 +20,7 @@ import { sendSms } from './single-send-text.js';
 import type { Template } from './types.js';
 import { aliyunSmsConfigGuard, sendSmsResponseGuard } from './types.js';
 
-const isChinaNumber = (to: string) => /^(\+86|0086|86)?\d{11}$/.test(to);
+const isChinaNumber = (to: string) => /^(\+86|0086|86)\d{11}$/.test(to);
```

Comment:
@logto-io/eng can you guys take a look?

Reply:
@wangsijie @darcyYe

Reply:
But the existing data may not have country code prefix, hold this PR until we have a plan for migration.